crates/sui-types/src/lib.rs

```diff
@@ -149,6 +150,18 @@ pub fn sui_framework_address_concat_string(suffix: &str) -> String {
     format!("{}{suffix}", SUI_FRAMEWORK_ADDRESS.to_hex_literal())
 }
 
```

Comment:
nit: add some doc comments, particularly when one shoud or should not use this function.